agent/agent-tooling/src/main/java/com/microsoft/applicationinsights/agent/internal/wasbootstrap/configuration/Configuration.java

```diff
@@ -570,6 +571,11 @@ public void validate() throws FriendlyException {
         public boolean enabled = false;
     }
 
+    public static class GcEventConfiguration {
+        public boolean reportAllGcEvents = false;
+        public boolean reportTenuredGcEvents = false;
```

Comment:
or maybe a single enum property, similar to logging `level`, e.g. `NONE`, `TENURED`, `ALL`?
```suggestion
        public boolean reportAll = false;
        public boolean reportTenured = false;
```